### DISCUSSION TITLE
Priority assignment for the components

### DISCUSSION THREAD

**Discussion by yuganshmohan** [nasa/fprime#913] [Q&A] [answered]
Hey, I've just started with F Prime and I wanted to know if there is a way to give priority to a component, for eg. can we give telemetry & command processing a higher priority than data processing workflow?

**Comment by Joshua-Anderson** [comment, marked as answer]
You set can set the priority of any activity component when you start it:

```
rateGroup1Comp.start(0, 120,10 * 1024)
```

```
void start(NATIVE_INT_TYPE identifier, NATIVE_INT_TYPE priority, NATIVE_INT_TYPE stackSize, NATIVE_INT_TYPE cpuAffinity = -1);
```

I believe that the lower the priority number the lower the task priority.
